src/platform/forms-system/src/js/widgets/PhoneNumberWidget.jsx

```diff
@@ -7,9 +7,12 @@ import TextWidget from './TextWidget';
  * instead
  */
 export default class PhoneNumberWidget extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = { val: props.value };
+  state = { val: this.props.value, firstUpdate: true };
+
+  componentDidUpdate(prevProps) {
```

Comment:
This hook might run once and only once if the component's props.value is modified.